ecs_grafana.tf

```diff
@@ -0,0 +1,93 @@
+resource "aws_ecs_task_definition" "grafana" {
+  count                    = local.is_management_env ? 1 : 0
+  family                   = "grafana"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "512"
+  memory                   = "4096"
+  task_role_arn            = aws_iam_role.prometheus[count.index].arn
+  execution_role_arn       = local.is_management_env ? data.terraform_remote_state.management.outputs.ecs_task_execution_role.arn : data.terraform_remote_state.common.outputs.ecs_task_execution_role.arn
+
+  container_definitions = <<DEFINITION
+[
+  {
+    "cpu": ${var.fargate_cpu},
+    "image": "${data.terraform_remote_state.management.outputs.ecr_grafana_url}",
+    "memory": ${var.fargate_memory},
+    "name": "grafana",
+    "networkMode": "awsvpc",
+    "user": "0:0",
+    "portMappings": [
+      {
+        "containerPort": 3000,
+        "hostPort": 3000
+      }
+    ],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${data.terraform_remote_state.management.outputs.ecs_cluster_main_log_group.name}",
```

Comment:
I think it needs it's own log group, rather than using the main cluster one

Reply:
It does, I need to do this for all the containers. So i've created a ticket DW-4386 that will address this.